backend/routes/ocr.js

```diff
@@ -1,21 +1,14 @@
 var express  = require('express');
 var router   = express.Router();
 var model    = require('../models/ocr.js');
+var multer  = require('multer');
+var upload = multer();
 
-router.post('/', function(req,res){
-  model.parseImage(req.body.url)
-    .then(function(output) {
-      res.status(200).send(output);
-    });
-});
-
-router.get('/', function(req,res){
-  console.log("THIS IS A DEBUGGING ROUTE");
-  var dummyUrl = 'http://cdn2.theweek.co.uk/sites/theweek/files/2016/03/16lup800pxwidea_0.jpg';
-  model.parseImage(dummyUrl)
-    .then(function(result) {
-      console.log(result);
-      res.status(200).send(result);
+router.post('/',upload.single('poster'), function(req,res){
+  model.parseImage(req.file)
+    .then(function(artistList) {
+      res.status(200).send(artistList);
+      //call playlist_creation(artistList) here.
```

Comment:
Removed the comment instead of calling the create playlist directly, since in the frontend we're actually waiting for our user to push the 'create playlist' button and not create it straight away :oops: